tests/baselines/reference/taggedTemplateStringsWithTypedTags.js

```diff
@@ -43,4 +43,4 @@ var f;
 (_j = ["abc", "def", "ghi"], _j.raw = ["abc", "def", "ghi"], (_k = ["abc", "def", "ghi"], _k.raw = ["abc", "def", "ghi"], f(_k, 1, 2))["member"].member(_j, 1, 2));
 f.thisIsNotATag("abc");
 f.thisIsNotATag("abc" + 1 + "def" + 2 + "ghi");
-var _a, _b, _c, _d, _e, _f, _g, _h, _j, _k;
+var _a, _b, _c, _d, _e, _f, _h, _g, _k, _j;

```

Comment:
The actual names aren't chosen until the printer runs, and the order temps are added in each transform isn't identical to the current emitter as we transform a file in phases.